src/server.cpp

```diff
@@ -2075,8 +2066,10 @@ void Server::SendActiveObjectRemoveAdd(RemoteClient *client, PlayerSAO *playersa
 			obj->m_known_by_count--;
 	}
 
-	if (!sounds_to_stop.empty())
-		stopAttachedSounds(client->peer_id, sounds_to_stop);
+	// Note: Do yet NOT stop or remove object-attached sounds for objects goes out of range
```

Comment:
"for objects goes out of range"

Reply:
My brain is apparently molten at this point.